### PR TITLE
New API: get method by name

### PR DESCRIPTION
**Pull request by Liyw979**

### CODE REVIEW

**Comment by JonasKlauke** [comment]
mmh I think getMethodByName would be a better name otherwise you might think you can input a string of a method signature

File: sootup.core/src/main/java/sootup/core/model/AbstractClass.java [hunk 1]
```diff
@@ -122,4 +123,15 @@ public Optional<? extends SootMethod> getMethod(
                   getClassSource().getSourcePath());
             });
   }
+
+  /**
+   * Attempts to retrieve the method with the given name. This method will return an empty Set if
+   * there is no method with the given name.
```
Comment (JonasKlauke) [inline sootup.core/src/main/java/sootup/core/model/AbstractClass.java:129]:
```suggestion
   * Attempts to retrieve the method with the given name. This method will return an empty Set if
   * there is no method with the given name.
   * @param name the name of the method
   * @return a set of methods that have the given name
```

**Comment by Liyw979** [comment]
> mmh I think getMethodByName would be a better name otherwise you might think you can input a string of a method signature

I think so.

**Review by JonasKlauke — APPROVED** [review]
👍

**Pull request merged**